cumulusci/tasks/release_notes/tests/test_task.py

```diff
@@ -23,9 +25,292 @@ def test_run_task(self, GithubReleaseNotesGenerator):
                 }
             ),
         )
+        project_config.project__git__default_branch = "master"
+        return project_config
+
+    @mock.patch("cumulusci.tasks.release_notes.task.GithubReleaseNotesGenerator")
+    def test_run_GithubReleaseNotes_task(
+        self, GithubReleaseNotesGenerator, project_config
+    ):
+        generator = mock.Mock(return_value="notes")
+        GithubReleaseNotesGenerator.return_value = generator
         task_config = TaskConfig({"options": {"tag": "release/1.0"}})
         task = GithubReleaseNotes(project_config, task_config)
         task.github = mock.Mock()
         task.get_repo = mock.Mock()
         task()
         generator.assert_called_once()
+
+
+class TestParentPullRequestNotes(GithubApiTestMixin):
+
+    BRANCH_NAME = "test-branch"
+
+    @pytest.fixture
+    def gh_api(self):
+        return get_github_api("TestOwner", "TestRepo")
```

Comment:
We can put reused pytest fixtures in a conftest.py module (either in one particular test folder or at the root of cumulusci) and then they'll be available to all tests without needing to redefine them.

Reply:
Done.